embulk-standards/src/main/java/org/embulk/standards/LocalFileInputPlugin.java

```diff
@@ -148,6 +165,13 @@ public FileVisitResult preVisitDirectory(Path path, BasicFileAttributes attrs)
                 @Override
                 public FileVisitResult visitFile(Path path, BasicFileAttributes attrs)
                 {
+                    try {
+                        if(  Files.isDirectory(path.toRealPath()) ) {
+                            return FileVisitResult.CONTINUE;
+                        }
```

Comment:
Can you explain and comment what you meant? It would be helpful for maintaining the code.